ports/alienpogs/port.json

```diff
@@ -11,8 +11,8 @@
     "porter": [
       "TheLaggyPyro"
     ],
-    "desc": "This Bullet Heaven Survival game was made for GBJam11 from Sept 15-25 2023. .",
-    "inst": "Download any version of the game from  [Itch.io](https://meredosia.itch.io/alien-pogs-from-outer-space), extract the zip file and copy the .rpg file into the the  alienpogs port folder.",
+    "desc": "This Bullet Heaven Survival game was made for GBJam11 from Sept 15-25 2023 .",
+    "inst": "Download the game from Itch(https://meredosia.itch.io/alien-pogs-from-outer-space), extract zip file and copy .rpg file into the alienpogs port folder.",
```

Comment:
By easy and short I meant download the game from Steam and done.  No Links since they're on their device and not a pc. Like short and easy no links or fuzz.